db/migrate/20210212160639_remove_published_status_from_content_page.rb

```diff
@@ -0,0 +1,5 @@
+class RemovePublishedStatusFromContentPage < ActiveRecord::Migration[6.1]
```

Comment:
As this is not in `main` I don't think you need this migration.

You might need to rebase with `main`, drop db, create db and run the existing migrations from the start to get your environment realigned.

Reply:
/cc @guy-roberts